python/ray/dashboard/modules/aggregator/tests/test_ray_job_events.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+import sys
+import json
+import base64
+
+import ray
+from ray._private.test_utils import (
+    wait_until_server_available,
+    wait_for_condition,
+)
+from ray.dashboard.tests.conftest import *  # noqa
+
+from ray._private import ray_constants
+
+
+@pytest.fixture(scope="session")
+def httpserver_listen_address():
+    return ("127.0.0.1", 12345)
+
+
+def test_ray_job_events(ray_start_cluster, httpserver):
+    cluster = ray_start_cluster
+    cluster.add_node(
+        env_vars={
+            "RAY_DASHBOARD_AGGREGATOR_AGENT_EXPOSABLE_EVENT_TYPES": "DRIVER_JOB_EVENT",
+        }
+    )
+    cluster.wait_for_nodes()
+    ray.init(address=cluster.address)
+
+    # Wait for the dashboard agent to be available
+    ip, _ = cluster.webui_url.split(":")
+    agent_address = f"{ip}:{ray_constants.DEFAULT_DASHBOARD_AGENT_LISTEN_PORT}"
+    assert wait_until_server_available(agent_address)
```

Comment:
we should remove this from the test, it could mask the fact that the system loses data during the startup sequence

Reply:
hm, I didn't realize that PR is related to events too, it only mentions metrics